rag/utils/llm.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from rag.utils.configurables import (
+    llm_modelname_configurable,
+    llm_temperature_configurable,
+)
+
+from langchain_openai import ChatOpenAI
+
+MODEL_NAME = os.environ.get("MODEL_NAME", "gpt-4o")
```

Comment:
We changed it back to gpt-3.5-turbo right? I guess that hasn't been pushed to this branch. It would be great to test out `gpt-4o-mini` here!

Reply:
When you make a request or use langserve remoterunnables, you can add the model name as config. Once I've added the styling on the frontend streamlit app you can test it out that way as well.

Reply:
Adding a note to update docs on how to use remote runnable with config